ee/localserver/request-query.go

```diff
@@ -0,0 +1,49 @@
+package localserver
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+func (ls *localServer) requestQueryHandler() http.Handler {
+	return http.HandlerFunc(ls.requestQueryHanlderFunc)
+}
+
+func (ls *localServer) requestQueryHanlderFunc(res http.ResponseWriter, r *http.Request) {
+	if r.Body == nil {
+		res.Write([]byte("request body is nil"))
+		return
```

Comment:
For all of these, I think we should be returning a status. 

It might be worth making a helper along the lines of `sendClientError(w http.ResponseWriter, message string)`

Reply:
done, returning bad request for errors